spec/ingram_micro/elements/sales_order_header_spec.rb

```diff
@@ -6,6 +6,7 @@
 
     context 'when using the domestic schema' do
       it "returns the correct default values hash" do
+        IngramMicro.configuration.international = false
```

Comment:
@sent-hil it's in the Configuration. That's a pretty easy change, will add it to my updates today.